### PR TITLE
Add chown to permissions page

### PR DESCRIPTION
**Pull request by hflynn**
See https://www.openmicroscopy.org/community/viewtopic.php?f=4&t=8381

The fact that group owners (and admins) can change the ownership of data was not included in the permissions table.

### CODE REVIEW

**Comment by pwalczysko** [comment]
Remove "now easily" from the https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR16 - "now" is very dated, and "easily" is not totally accurate (the action is easy to trigger, but the consequences are non-trivial under circumstances)

**Comment by pwalczysko** [comment]
In https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR50, "This would typically be a PI..." refers to Group Owner. But as the previous sentence is also mentioning administrators, this is not completely clear.

**Comment by pwalczysko** [comment]
In https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR79 "...as somewhere to store their data..." - maybe a bit clumsy formulation, why "somewhere"

**Comment by pwalczysko** [comment]
This https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR81 "...copied..." strongly suggests there is such thing as copy between groups in OMERO. But there is not. Maybe change to "moved"

**Comment by pwalczysko** [comment]
In https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR91
- remove "some" in "...some annotations..." - all possible annotations are possible to do for Group Owner in this situation.

**Comment by pwalczysko** [comment]
after this line https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR100 I would add a bullet point saying something like:

``Read-only group is a part of the Publication workflow in OMERO, see http://help.openmicroscopy.org/publish.html#public``

**Comment by pwalczysko** [comment]
Removed my previous comment about publications and Read-only. The statement is there in the next bullet point. But I would still cite http://help.openmicroscopy.org/publish.html#public
in https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR104

**Comment by pwalczysko** [comment]
I am not sure about the ``intermediate`` in ``...This is an intermediate option...``. We have four group levels, thsi is the second one. https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR87

**Comment by pwalczysko** [comment]
``..a more collaborative group.`` Maybe ``...a group where some collaboration on other members data is enabled`` https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR107

**Comment by pwalczysko** [comment]
In here https://github.com/openmicroscopy/ome-documentation/pull/1794/files#diff-3c1d6113c0b3e790090e598763e385caR297 there is a missing comma before ``ROIs``

File: omero/sysadmins/server-permissions.rst [hunk 1]
```diff
@@ -307,11 +311,11 @@ Key
         objects such as images.
 
     Move between groups
-        Only the admin has the right to move other users’
+        Only an admin has the right to move other users’
         data between groups.
 
         .. note::
-            The admin does not have to be member of the
+            An admin does not have to be member of the
```
Comment (pwalczysko) [inline omero/sysadmins/server-permissions.rst]: An admin does not have to be member of neither the original, nor the destination group.

**Comment by pwalczysko** [comment]
Finished reading, couple of reformulation suggestions in the whole doc.

**Comment by hflynn** [comment]
@pwalczysko I think I've caught all of those in my latest commit, I'll rebuild the merge docs once Travis is green.

**Comment by hflynn** [comment]
https://ci.openmicroscopy.org/view/Docs/job/OMERO-DEV-merge-docs/886/ includes the latest changes here

File: omero/sysadmins/server-permissions.rst [hunk 2]
```diff
@@ -98,13 +98,18 @@ The various permission levels are:
          other group members to access and view their data. Their PI, as a
          group owner could then annotate and/or add Regions of Interest
          (ROIs) to their images.
+      -  Scientists submitting a publication could move data to a read-only
+         group as part of the publication workflow, making them publicly
+         available via a URL for reviewers and readers (see the
+         :help:`guide guide for public data <publish.html#public>`)
```
Comment (pwalczysko) [inline omero/sysadmins/server-permissions.rst]: Typo: ``guide guide`` -> remove one ``guide``

**Comment by pwalczysko** [comment]
One typo found https://github.com/openmicroscopy/ome-documentation/pull/1794#discussion_r145916654. 

After fixing the typo, can be merged. Thanks @hflynn

**Pull request merged**